save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/HasErrorModal.kt

```diff
@@ -25,7 +25,7 @@ import web.html.ButtonType
  */
 @Suppress("MAGIC_NUMBER")
 val ringLoader = ringLoader(jso {
-    this.size = 70
+    this.size = 80
```

Comment:
What does `80` mean? 

I mean, unit-wise?

Reply:
I am think, that it's px )